### PR TITLE
Changed absolute links to relative links

### PR DESCRIPTION
**Pull request by bdamerac**
adds relative links, addressing #222

### CODE REVIEW

**Comment by evykassirer** [comment]
beautiful!! much better

**Comment by evykassirer** [comment]
can you add "Addresses #222" to the description?

File: content/S2016/S16-event-machine-learning.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ Technical Workshop Series: This event is generously sponsored by
 Technical Workshop Series. We are hosting three such workshops 
 focused on teaching technical skills in partnership with BlackBerry. 
 To learn more about our Technical Workshop Series and partnership 
-with BlackBerry see our [sponsors page](http://wics.uwaterloo.ca/sponsorship/).
+with BlackBerry see our [sponsors page]({filename}/sponsorship.md).
```
Comment (evykassirer) [inline content/S2016/S16-event-machine-learning.md]: pretty sure there shouldn't be the md at the end, but we can test it on the sample site

Comment (evykassirer) [inline content/S2016/S16-event-machine-learning.md]: OH nvm I gave you the wrong advice

it's `pages/sponsors.md` cause that's what the actual file is named in the pages folder

Comment (bdamerac) [inline content/S2016/S16-event-machine-learning.md]: Okay!
Will change that soon

File: content/S2016/S16-event-machine-learning.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ Technical Workshop Series: This event is generously sponsored by
 Technical Workshop Series. We are hosting three such workshops 
 focused on teaching technical skills in partnership with BlackBerry. 
 To learn more about our Technical Workshop Series and partnership 
-with BlackBerry see our [sponsors page](http://wics.uwaterloo.ca/sponsorship/).
+with BlackBerry see our [sponsors page]({filename}/pages/sponsorship.md).
```
Comment (evykassirer) [inline content/S2016/S16-event-machine-learning.md]: s/sponsorship/sponsors

Comment (bdamerac) [inline content/S2016/S16-event-machine-learning.md]: Fixed it, can assure it works :)

Comment (evykassirer) [inline content/S2016/S16-event-machine-learning.md]: woohoo, thanks for checking it too

did you go through the rest of the links to double check they work?

**Comment by bdamerac** [comment]
All the links that have been modified work.

**Comment by fboxwala** [comment]
This looks good to me :) If there are no other comments will merge later today

**Comment by claricen** [comment]
looks good to me too :)

**Pull request merged**